CONTRIBUTORS.md

```diff
@@ -53,7 +53,7 @@
 51. [Awosise Oluwaseun](#awosise-oluwaseun)
 52. [Issac Jacob](#issac-jacob)
 53. [Adeola Adetunji](#adeola-adetunji)
-
+54. [Ndi Lionel](#spykelionel)
```

Comment:
Please hyperlink your name with the H2 heading you added above your detailed bio

Reply:
Added hyperlink to name